src/gui/states.py

```diff
@@ -138,7 +138,7 @@ def on_event(self, event):
         elif event_type == Event.PIN_CODE_REQUESTED:
```

Comment:
This implementation and event can be removed. This was implemented when we thought that we would have a function for requesting PIN codes from with in member booth.

Reply:
I am removing it from the branch you are trying to commit to.